iroh/src/magicsock/relay_actor.rs

```diff
@@ -346,6 +405,12 @@ impl RelayActor {
                     trace!("shutting down");
                     break;
                 }
+                Some(Err(err)) = self.active_relay_tasks.join_next() => {
+                    if !err.is_cancelled() {
```

Comment:
needs panic check

Reply:
Does it?  I explicitly only check for anything **not** cancelled so it includes panics.  My first version indeed had `if err.is_panic()` but the error handling in this case was exactly the same: because I don't think propagating a panic up here will help much.

Or do you think panics should **always** be propagated upwards?

Reply:
> Or do you think panics should **always** be propagated upwards?

this is currently what we do in all other cases, so for consistency I think that is the best way to do so for now